server/consumer.go

```diff
@@ -1199,6 +1202,7 @@ func (o *Consumer) Info() *ConsumerInfo {
 		NumAckPending:  len(o.pending),
 		NumRedelivered: len(o.rdc),
 		NumPending:     o.sgap,
+		Cluster:        o.srv.clusterInfo(o.node),
```

Comment:
I understand that the PR is on hold, but just a quick note: this will result in getting the server lock, under the consumer lock. Not sure if this is correct.

Reply:
Thanks will take a look.

Reply:
ok fixed.